udf/server_test.go

```diff
@@ -337,6 +337,7 @@ func TestUDF_SnapshotRestore(t *testing.T) {
 		req = <-u.Requests
 		if req == nil {
 			t.Error("expected snapshot message got nil")
+			return
```

Comment:
Alternatively, you could do t.Fatal(...)

Reply:
Not supposed to do a t.Fatal in a goroutine that isn't the main one.